MetaLamp/lending-pool/test/Spec/User.hs

```diff
@@ -0,0 +1,140 @@
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications  #-}
+
+module Spec.User where
+
+import           Control.Lens               ((^?))
+import           Control.Monad              (forM, void)
+import qualified Data.Map                   as Map
+import           Data.Monoid                (Last (..))
+import           Data.Text                  (Text)
+import           Data.Void                  (Void)
+import qualified Ledger
+import           Plutus.Contract            hiding (throwError)
+import           Plutus.Contract.Test
+import qualified Plutus.Contracts.AToken    as AToken
+import qualified Plutus.Contracts.Core      as Aave
+import           Plutus.Contracts.Endpoints (ContractResponse (..))
+import qualified Plutus.Contracts.Endpoints as Aave
+import           Plutus.PAB.Simulation      (initContract, toAsset)
+import qualified Plutus.Trace.Emulator      as Trace
+import           Plutus.V1.Ledger.Ada       (lovelaceValueOf)
+import           Plutus.V1.Ledger.Value     (AssetClass, Value, assetClassValue)
+import qualified PlutusTx.AssocMap          as AssocMap
+import qualified PlutusTx.Prelude           as PlutusTx
+import qualified Spec.Mock                  as Mock
+import qualified Spec.Utils                 as TestUtils
+import           Test.Tasty
+
```

Comment:
We can create a module for each specific use case that would have only all test cases described, and another module for test fixtures